staging/src/kubevirt.io/client-go/api/v1/schema.go

```diff
@@ -32,6 +32,7 @@ const (
 	IOThreadsPolicyAuto    IOThreadsPolicy = "auto"
 	CPUModeHostPassthrough                 = "host-passthrough"
 	CPUModeHostModel                       = "host-model"
+	SchedulerFIFO                          = "fifo"
```

Comment:
does this belong in the VMI api's schema.go file if it's only used internally during the VMI spec to domain spec conversion in virt-launcher? 

Maybe it makes more sense in `pkg/virt-launcher/virtwrap/api/schema.go ` for now?

Reply:
Sounds good.